app/styles/main.scss

```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```

Comment:
Part of what I'm getting at is that if the footer height changes in the future (someone adds another social link, etc) then we end up having to modify this CSS to make it not break.

I think maybe the complexity is higher than the benefit.

Thoughts?